src/formats/perkinElmer.js

```diff
@@ -0,0 +1,33 @@
+export const parsePerkinElmerChannels = (images) => {
```

Comment:
Can you explain what this is, and what it is doing? Does it need attribution of any type?

Reply:
No this was custom, based on the qptiff spec. I could attribute [the OME docs](https://docs.openmicroscopy.org/bio-formats/6.5.1/formats/perkinelmer-vectra-qptiff.html), this has a download for the format spec. 

qpTIFF images always have an XML doc in their image description, which in turn always have a name and color of the channel. Here I take the first set of images after sorting by aspect ratio, and group them by channel

Reply:
Might be worth at least linking to that in a comment for future reference.